### ISSUE TITLE
network driver on pummelzacken broken

### ISSUE THREAD

**Issue by lonvia** [openstreetmap/operations#45]
Since the reboot of pummelzacken on 23rd around 2am, the igb module reports a hanging TX unit and continuously resets the network card:

~~~
[  892.446179] igb 0000:02:00.1: Detected Tx Unit Hang
[  892.446179]   Tx Queue             <7>
[  892.446179]   TDH                  <a>
[  892.446179]   TDT                  <a>
[  892.446179]   next_to_use          <c>
[  892.446179]   next_to_clean        <a>
[  892.446179] buffer_info[next_to_clean]
[  892.446179]   time_stamp           <100023a25>
[  892.446179]   next_to_watch        <ffff88102755d0b0>
[  892.446179]   jiffies              <1000241b4>
[  892.446179]   desc.status          <13a4200>
[  894.457627] igb 0000:02:00.1 em2: Reset adapter
[  894.484937] igb 0000:02:00.0 em1: Reset adapter
~~~

I've already tried [this advice](https://communities.intel.com/message/212871#212871) but with no success.

**Comment by zerebubuth** [comment]
Hmm... Aug 23 01:51:56 was the first time it booted 3.16.0-47 kernel, and the [diff to -46](https://launchpad.net/ubuntu/+source/linux-lts-utopic/3.16.0-47.63~14.04.1) makes a reference to [this bug](https://bugs.launchpad.net/ubuntu/+source/linux/+bug/1454892) which is the only networking-related thing I could find. Do you think that it's worth trying the older kernel, or a much newer one? My gut feeling is that the correlation between the reboot to a newer kernel and this problem appearing isn't a coincidence.

**Comment by lonvia** [comment]
Rebooting into the already available 3.16.0-48 didn't help. Going back to the previous one would be a good idea. Looks like it was running 3.16.0-46 before.

**Comment by lonvia** [comment]
Reverting to 3.16.0-46 kernel resolved the problem.
